egs/sre08/v1/sid/nnet3/xvector/allocate_egs_seg_v1.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python3
+
+# Copyright 2017 Johns Hopkins University (author: Hang Lyu)
+
+"""This script is suitable for the training set whose size is not very big, such
+as TS learning, age or emotion id.
```

Comment:
Try to combine allocate_egs_seg_v*.py into allocate_egs_seg.py! It is better to have single script.
would you describe exact differences among them?